ftest.sh

```diff
@@ -294,7 +294,7 @@ wq
 EOF
 fi
 
-pushd src/tests/ftest
+pushd install/lib/daos/TESTING/ftest
```

Comment:
Ditto.

Reply:
This line change is the 1st step that moves CI test to use the newly located avocado tests, next step will be addressed by:
https://jira.hpdd.intel.com/browse/DAOS-3607
Will move CI test to DAOS RPM env.